### PR TITLE
Feat/issue 45 orderstate checkinpage

### PR DESCRIPTION
**Pull request by hellodebs**
hello, 

- I did some random renaming of my confirm paths
- after confirming the order, the order state resets to 0
- I created the check in page. You are actually able to scan a QR code, which guides you to the menu page.

### CODE REVIEW

File: src/components/CheckIn.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+import { useEffect } from "react";
+
+export default function CheckIn({ updateTableId }) {
+  let history = useHistory();
+  const { tableId } = useParams();
+  useEffect(() => {
+    updateTableId(tableId);
+    history.push("/menu");
+  });
+  return <></>;
```
Comment (namirsab) [inline src/components/CheckIn.js:11]: Interseting solution! If you don't want to render anything, you can as well return null

**Review by namirsab — APPROVED** [review]
@hellodebs looking good, interesting solution for getting the table id. I like it :)

File: src/App.js [hunk 2]
```diff
@@ -6,21 +6,22 @@ import usePersistedState from "./hooks/usePersistedState";
...
 function App() {
   const [menu, setMenu] = useState([]);
   const [order, setOrder] = usePersistedState("order", []);
+  const [tableId, setTableId] = usePersistedState("table", 0);
 
   let history = useHistory();
 
   useEffect(() => {
-    const url = "/api/menu.json";
+    const url = `/api/menu.json`;
```
Comment (johnxgaehner) [inline src/App.js:24]: why did you change it to ``? you are using "" everywhere else. :)

File: src/App.js [hunk 3]
```diff
@@ -71,44 +72,52 @@ function App() {
...
   function billButtonHandler() {
     if (window.confirm("Would you like to receive the bill?")) {
-      history.push("/ConfirmBillText");
+      history.push("/confirm-bill-text");
     }
   }
 
   return (
     <Div100vh className="App">
       <header className="App__header">
```
Comment (johnxgaehner) [inline src/App.js:100]: would look nice if you move the header into its own component, I think. :)

**Review by johnxgaehner — APPROVED** [review]
final sprint! let's go!

apart from that pull request, I noticed that you still show 4.5 € and not 4.50 € and when you give the menu__counter--number a width it would be more easy to in/decrease because the buttons next to it are always in the same position and don't move.

File: src/App.js (hunk 2 above)
Comment (hellodebs) [reply src/App.js:24]: Actually I tried something else there and just didn't change it back 😄  haha

File: src/App.js (hunk 3 above)
Comment (hellodebs) [reply src/App.js:100]: Something I listed in my fine cut ;)
